readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://<natispatis>.github.io/layout_html-form/)
```

Comment:
Think here you should remove brackets <> otherwise links will not work correctly, same for testing